src/components/Switch/_Switch.scss

```diff
@@ -0,0 +1,133 @@
+.switch {
```

Comment:
1. Redesign to Figma 
2. Use pat-ui color convention
3. Use a couple of examples of mixins
4. Use SCSS variables 
5. Increase the speed of the component similar to material-ui